packages/machine-extractor/src/extractAction.ts

```diff
@@ -293,23 +231,18 @@ export function extractSendToAction(
             t.isObjectProperty(prop) && t.isIdentifier(prop.key),
         )
         .forEach((prop) => {
+          const name = prop.key.name;
           if (t.isStringLiteral(prop.value) || t.isBigIntLiteral(prop.value)) {
-            options[prop.key.name] = {
-              type: 'string',
-              value: prop.value.value,
-            };
+            options[name as 'id' | 'delay'] = prop.value.value;
```

Comment:
We don't have the luxury of trusting this input like that. We should filter out properties that are outside of this handled set